pkg/http/core/applications.go

```diff
@@ -761,8 +805,17 @@ func listServerGroups(c *gin.Context, wg *sync.WaitGroup, sgs chan ServerGroup,
 func makeServerGroupMap(pods *unstructured.UnstructuredList) map[string][]Instance {
 	// Map of server group to instances (pods)
 	serverGroupMap := map[string][]Instance{}
+	items := pods.Items
+	// Sort the pods.
+	sort.Slice(items, func(i, j int) bool {
+		if items[i].GetNamespace() != items[j].GetNamespace() {
+			return items[i].GetNamespace() < items[j].GetNamespace()
+		}
+
+		return items[i].GetName() < items[j].GetName()
```

Comment:
Why are we using GetName here instead of just Name? Like to good above.

Reply:
Because in this case, `items` is of type `unstructured.Unstructured` which does not have a `Name` field, but does have a `GetName()` function.